docs/02-app/02-api-reference/04-functions/generate-metadata.mdx

```diff
@@ -1041,7 +1041,7 @@ ReactDOM.prefetchDNS(href: string)
 >
 > - These methods are currently only supported in Client Components, which are still Server Side Rendered on initial page load.
 > - Next.js in-built features such as `next/font`, `next/image` and `next/script` automatically handle relevant resource hints.
-> - React 18.3 does not yet include type definitions for `ReactDOM.preload`, `ReactDOM.preconnect`, and `ReactDOM.preconnectDNS`. You can use `// @ts-ignore` as a temporary solution to avoid type errors.
+> - React 18.3 does not yet include type definitions for `ReactDOM.preload`, `ReactDOM.preconnect`, and `ReactDOM.prefetchDNS`. You can use `// @ts-ignore` as a temporary solution to avoid type errors.
```

Comment:
This whole line is obsolete now. The types have been available for a while now.

Reply:
@eps1lon Oh wow, great catch, thanks. 👍 I've now made those amendments 👉 https://github.com/vercel/next.js/pull/65111/commits/312b559d5ae51a7c591f129f7f599433d4b84a79